files/zipgateway.cfg

```diff
@@ -36,3 +36,4 @@ ZWRFRegion=0x00
 #Measured0dBmPower=
 #ZipDeviceID=AABBCCDDEEFF
 #MaxLRTxPowerLevel=
+TimeResendSerial=50
```

Comment:
```suggestion
#TimeResendSerial=50
```

Any reason to do double affectation here ? I would keep it commented if it will be set by code